omaha/base/time_unittest.cc

```diff
@@ -141,8 +141,7 @@ TEST(TimeTest, RFC822TimeParsing) {
   ASSERT_EQ(time.wYear , 2005);
   ASSERT_EQ(time.wMonth , 5);
   ASSERT_EQ(time.wDay , 16);
-  //ASSERT_TRUE(time.wHour == 15 || time.wHour == 14);  // daylight saving time
-  ASSERT_TRUE(time.wHour == 18 || time.wHour == 17);  // EST/EDT
+  ASSERT_TRUE(time.wHour == 15 || time.wHour == 14);  // daylight saving time
```

Comment:
I rolled this back and allowed the test to fail, since this isn't upstream able; I've added a list of disabled tests to omaha-build, including this one.